### DISCUSSION TITLE
Replenish Nightscout data in case of Dexcom server failure

### DISCUSSION THREAD

**Discussion by DaNitroo** [NightscoutFoundation/xDrip#2759] [Q&A]
Hello,

the dexcom servers were down for several hours last night. We use the Dexcom receiver and also a cell phone and both devices were refilled after an hour and are now fully displayed again.

But neither in XDrip nor in Nightscout was this data gap filled. XDrip and Nightscout are both set up as Dex Share Followers and I thought that the data would be automatically filled in there, but unfortunately that doesn't happen :-(

Does anyone have an idea?

**Comment by Navid200** [comment]
What uploads to Nightscout in this scenario?

**Reply by DaNitroo** [reply]
The blood sugar, i.e. the measured values. All measured values ​​are available in the mobile phone and in the Dexcom receiver and can be read off. However, since the Dexcom server connection was not ok for several hours, the blood glucose values ​​were not transferred and we cannot see them in Nightscout / XDrip.
